assets/app/view/game/bid.rb

```diff
@@ -8,14 +8,16 @@ module Round
       class Bid < Snabberb::Component
         include Actionable
         needs :entity
-        needs :corporation
+        needs :corporation, default: nil
+        needs :company, default: nil
```

Comment:
Generalize corporation and company into one type.

Reply:
Sure, im have mostly programmed in strong type languages. Not used to use the same variable for different objects.